src/styles/main.scss

```
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./reset";
+@import "./normalize";
+@import "./utils/variable";
+@import "./utils/extends.scss";
```

```suggestion
@import "./utils/extends";
```

You can omit extension in imports